/*
5.0  Create a class called Date that includes three instance variables—a month (type int), a day (type int) and a year (type int). Provide a constructor that initializes the three instance variables and assumes that the values provided are correct. Provide a set and a get method for each instance variable. Provide a method displayDate that displays the month, day and year separated by forward slashes (/). Write a test application named DateTest that demonstrates class Date’s capabilities. 
*/

class Date{
    int month, year, day;
    Date(int month, int day, int year){
        this.month = month;
        this.day = day;
        this.year = year;
    }
    public void setMonth(int month){
        this.month = month;
    }
    public int getMonth(){
        return month;
    }
    public void setYear(int year){
        this.year = year;
    }
    public int getYear(){
        return year;
    }
    public void setDay(int day){
        this.day = day;
    }
    public int getDay(){
        return day;
    }
    public void displayDate(){
        System.out.println(getDay()+"/"+getMonth()+"/"+getYear());
    }
}
public class q5 {
    public static void main(String[] args) {
        Date d = new Date(11, 24, 2026);
        d.displayDate();
    }
}
/*
24/11/2026
*/